### DISCUSSION TITLE
Some love for the frontend

### DISCUSSION THREAD

**Discussion by subjacked** [pkuehnel/TeslaSolarCharger#1106] [Ideas]
Hi,

I would like to contribute in the frontend department of TSC, as I really love it’s functions. The Design gets the Job done but could need some love. 

What would you guys want to see? 

Best regards,

Sebastian

**Comment by pkuehnel** [comment]
Well, as I am a backend developer and do not see the issues, I suggest improving what you want to improve. But I would like you to use as many [MudbBlazor ](https://mudblazor.com/docs/overview) components as possible. When I introduced the Backup and restore feature, I started using it. For example, there is also a [Menu bar](https://mudblazor.com/components/navmenu#basic-usage) and many other components that could improve the frontend, like [charts](https://mudblazor.com/components/barchart),(e.g. for spot price charge cost visualization), [Tables](https://mudblazor.com/components/table#default-table),... I am not talking about spacing and so on; Bootstrap is fine for that; I am only talking about components.

**Reply by subjacked** [reply]
I will setup the dev environment and look what options I have. Would love to see charts on the dashboard. We could visualize the current pv power with a gradient, so you see on a glance that your Cars can charge. Many ideas. I will use whatever you decide, don't want to destroy your base.

**Reply by pkuehnel** [reply]
Thanks for your help, if you need any help, just tell me
